packages/sdk/src/sound/setSoundStateOptions.ts

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Parameters to the `Sound.setSoundState` and `Actor.playSound` calls.
+ */
+export type SetSoundStateOptions = {
+    /**
+     * enabled (pause/resume).
+     */
+    enabled?: boolean,
+
+    /**
+     * pitch offset in halftones (0=default, 12=one octave higher, -12=one octave lower)
+     */
+    pitch?: number,
+
+    /**
+     * volume modifier, measured in DB (0=default,10=10x power, -10=1/10 power.
+     */
+    volume?: number,
+
+    /**
+     * time in milliseconds
```

Comment:
>milliseconds [](start = 15, length = 12)

In the animation system, time is expressed in seconds. I'd like to be consistent across the API and standardize on seconds unless there's a compelling reason for milliseconds here.

Reply:
good point

---
In reply to: [256951167](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/191#discussion_r256951167) [](ancestors = 256951167)

Reply:
Resolved. I moved time to be on PlaySound/LoadAndPlaySound, rather than here,. but changed to be seconds

---
In reply to: [256951973](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/191#discussion_r256951973) [](ancestors = 256951973,256951167)